contributing.md

```diff
@@ -1,18 +1,26 @@
 
 # Contributing
 
-### Terms
+1. [How to contribute? Step by step guide](#how-to-contribute?-step-by-step-guide)
+2. [PR conventions](pr-conventions)
```

Comment:
Hey, I think the `PR conventions` would give `404`, you need to add the hyperlink as you did for `How to contribute?`. Same for the rest of the index values.
And maybe call this index section something like `Index`, whay say?